guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
<blockquote>Now, to have the high resolution output from the model, we need to have a decoder which is has learnable parameters and can take input the high resolution feature maps as the input.</blockquote><p> </p><p>In order to have high-resolution output from the model, we need to have a decoder that has learnable parameters and can take high-resolution feature maps as input.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>done.</p>

Reply:
<p>done.</p>